src/parkapi_sources/converters/vrs_p_r/vrs_p_r.py

```diff
@@ -41,15 +45,53 @@ class VrsParkAndRidePushConverter(NormalizedXlsxConverter):
         # Other opening times are there, but not parsable
     }
 
+    def handle_xlsx(self, workbook: Workbook) -> tuple[list[StaticParkingSiteInput], list[ImportParkingSiteException]]:
+        worksheet = workbook.active
+        mapping: dict[str, int] = self.get_mapping_by_header(next(worksheet.rows))
+        row_values = [cell.value for cell in next(worksheet.rows)]
+
+        static_parking_site_errors: list[ImportParkingSiteException] = []
+        static_parking_site_inputs: list[StaticParkingSiteInput] = []
+
+        for row in worksheet.iter_rows(min_row=2):
+            # ignore empty lines as LibreOffice sometimes adds empty rows at the end of a file
+            if row[0].value is None:
+                continue
+
+            parking_site_dict = self.map_row_to_parking_site_dict(mapping, row)
+
+            if 'Zufahrt' in row_values:
```

Comment:
With this approach, you have a lot of unnecessary copy paste code. Why don't you add these lines in `map_row_to_parking_site_dict()`?

Reply:
I tried it but realised that in ```map_row_to_parking_site_dict()``` I had no access to list of original column names like ```Zufahrt``` in ```[cell.value for cell in next(worksheet.rows)]```. Is there a possibility to call ```worksheet.rows``` or list of original Excel column names in ```map_row_to_parking_site_dict()```?

Reply:
You could add this to `map_row_to_parking_site_dict` as a parameter, and add `**kwargs` to all other usages of this function to maintain the signature. This would make things easier for other modifications in future, too.

Reply:
Looks good